ipa-core/src/protocol/ipa_prf/mod.rs

```diff
@@ -410,14 +426,21 @@ pub mod tests {
                 test_input(10, 12345, true, 0, 5),
                 test_input(0, 68362, false, 1, 0),
                 test_input(20, 68362, true, 0, 2),
-            ];
+            ]; // trigger value of 2 attributes to earlier source row with breakdown 1.
```

Comment:
Do we really need to add this comment? If so, why not add another one above for the other one, which attributes to a source row with breakdown 2?

Reply:
I think it could be helpful, especially for folks still getting familiar with how IPA works overall.   I'll extend the comment to cover the other attributed pair.